README.md

```diff
@@ -20,24 +22,30 @@ Each RFC captures technical details for a specific change within a limited syste
 ## Workflow
 
 1. Product teams define product requirements in the product requirements doc (PRD).
-2. An internal O(1) Labs team member creates a pull request using the [0000-template.md](0001-template.md).
-3. The RFC describes changes in enough detail to inform the decision. This includes links to proposed changes to persistent or cannonical design records such as specs and code comments.
+2. An internal O(1) Labs team member creates the RFC pull request using the [0000-template.md](0001-template.md).
+3. The RFC submitter describe changes in enough detail to inform the decision. Be sure to include links to proposed changes to persistent or cannonical design records such as specs and code comments.
 4. Discussions occur in the RFC pull request within a rapid, defined time period.
-5. If architecture changes proposed in the initial RFC are significantly changed during the discussion, the submitter updates the RFC in a separate pull request.
+5. If architecture changes proposed in the initial RFC are significantly changed during the discussion, update the RFC in the open pull request. If the RFC PR is already merged, the submitter updates the RFC in a separate pull request.
 6. All architects must respond to the RFC pull request within two business days.
-7. RFC approval by all architects is required before implementation decisions.
+7. To keep the RFC current and alive, the submitter and product manager work together to get approval and closure on the RFC.
+8. RFC approval by all architects is required before implementation decisions.
 
 ## Qualities of effective RFCs
 
 - Prefer timely over perfect
 - Enable collaborative code comments, questions, and responses that capture the relevant RFC discussion
 - Provide context for changes that impact different product areas
+- Currency
```

Comment:
```suggestion
- Currency: RFC document is up-to-date with current product versions
```

Reply:
@mrmr1993  this point is to list up-to-date content as a desirable feature of an effective RFC

imo, this currency quality is implied, but we do have several open RFCs that might no longer be accurate for the current releases